b2/console_tool.py

```diff
@@ -1043,7 +1043,12 @@ class CreateBucket(DefaultSseMixin, Command):
     @classmethod
     def _setup_parser(cls, parser):
         parser.add_argument('--bucketInfo', type=json.loads)
-        parser.add_argument('--corsRules', type=json.loads)
+        parser.add_argument(
+            '--corsRules',
+            type=json.loads
+            help=
+            "If given, the bucket will have a 'custom' CORS configuration. Accepts a JSON formatted string."
```

Comment:
```suggestion
            "If given, the bucket will have a 'custom' CORS configuration. Accepts a JSON string."
```
it doesn't need to be "formatted", right? I think it would be better to specify it as "JSON string", otherwise people will may think you have to do the `.indent()` thing on it before you pass it here.

Reply:
Works for me.  To resolve my confusion the important bit of information was that it's a JSON _string_ and not a path to a JSON _file_ :)